libraries/botbuilder-middleware-aws-comprehend/src/engine.ts

```diff
@@ -1,28 +1,33 @@
 import { Engine } from '@botbuildercommunity/middleware-engine-core';
-import * as aws from 'aws-sdk';
+import { AWSComprehendOptions } from './schema';
+import { AWSError, Comprehend } from 'aws-sdk';
 
 /**
  * @module botbuildercommunity/middleware-aws-comprehend
  */
 
 export class AWSComprehendEngine extends Engine {
-    public client: aws.Comprehend;
-    public constructor() {
+    public client: Comprehend;
+    public options: AWSComprehendOptions;
+    public lang: string;
+    public constructor(options?: AWSComprehendOptions) {
         super();
-        this.client = new aws.Comprehend({ apiVersion: '2017-11-27', region: 'us-east-1' });
+        this.options = options || { apiVersion: '2017-11-27', region: 'us-east-1' };
+        this.lang = this.options.lang || 'en';
+        this.client = new Comprehend({ apiVersion: this.options.apiVersion, region: this.options.region });
     }
     public async entities(input: any): Promise<any> {
         const params = {
-            LanguageCode: 'en',
+            LanguageCode: this.options.lang,
```

Comment:
In the future it would be great to use the language from the context.activity by default, since it is not possible to instantiate a language dynamically now for this middleware. This could be great for multilingual bots, but this requires a change in the Middleware Engine core most likely, so I would just open this as a feature request for now. Not sure if this need is actually there.

Reply:
We need better language configuration across all packages. Would be a good project up for grabs. I’ll put an issue in for it.